src/services/sessions.js

```diff
@@ -1119,16 +1119,7 @@ module.exports = class SessionsHelper {
 					responseCode: 'CLIENT_ERROR',
 				})
 			}
-			// check if the session is accessible to the user
-			let isAccessible = await this.checkIfSessionIsAccessible(session, userId, isAMentor)
 
```

Comment:
Why was this code commented out?
@Prajwal17Tunerlabs

Reply:
"if the session was created by session manager then no need to check the accessibility of session to mentees, directly enrol all the mentees" was the task, so i removed it so that all mentees can now join